### ISSUE TITLE
Upgrade istio to 1.3

### ISSUE THREAD

**Issue by piotrmsc** [kyma-project/kyma#5315]
**Description**

In istio 1.3 pilot agent will have `/quitquitquit` handler released which will allow us to move on with k8s jobs having sidecars on azure. The problem is known from #4719. 

Before upgrading make sure it is indeed released (anything can happen ;-) ) So either look for commit in changelog or deploy on a minikube pure charts quickly, deploy app with sidecar, exec into istio-proxy and make a call to localhost:$PORT/quitquitquit

**Reasons**

Unblock failing tests jobs on Azure

**Attachments**

**Comment by Demonsthere** [comment]
The upgrade is temporarily halted due to a change in SDS handling in Istio. A bug report has been created: https://github.com/istio/istio/issues/17409

**Comment by piotrmsc** [comment]
1.3.4 will contain fix for that, we should update to it as soon as released

**Comment by piotrmsc** [comment]
Fix with sds should improve stablity of ingressgateway itself which give from time to time errora : https://github.com/kyma-project/kyma/issues/6005

**Comment by Demonsthere** [comment]
New PR: https://github.com/kyma-project/kyma/pull/6200
